prow/gerrit/fakegerrit/fakegerrit.go

```diff
@@ -70,6 +72,10 @@ func (fg *FakeGerrit) GetChangesForProject(projectName string, start, desiredTot
 	return res
 }
 
+func makeTimeStamp(t time.Time) gerrit.Timestamp {
```

Comment:
nit: not entirely sure that it worths a function, inline `gerrit.Timestamp{Time: t}` doesn't seem too bad to me